### PR TITLE
Allows waydroid to create PTY

### PR DESCRIPTION
**Pull request by Quackdoc**
doing this will allows LXC to create their own PTY instead of mounting host, this should be more secure then mounting /dev/pty 

addresses 

#145 #238

### CODE REVIEW

**Comment by Quackdoc** [comment]
10 is just an arbitrary number, I don't see any user exceeding this.

File: data/configs/config_2 [hunk 1]
```diff
@@ -19,6 +19,7 @@ lxc.net.0.hwaddr = 00:16:3e:f9:d3:03
 lxc.net.0.mtu = 1500
 
 lxc.console.path = none
+xc.pty.max = 10
```
Comment (aleasto) [inline data/configs/config_2:22]: typo, but also this would just be duplicate.

Comment (Quackdoc) [reply data/configs/config_2:22]: oops, but doesn't this need to be reflected in both files?

Comment (aleasto) [reply data/configs/config_2:22]: Sorry, that's right. I thought these were fragments then merged into a single config file.

**Comment by Etaash-mathamsetty** [comment]
can this get merged asap please, would like this fixed

**Comment by JamiKettunen** [comment]
This broke launching `waydroid-container` on Ubuntu Touch 16.04 (soon to be retired in favor of 20.04 where there is no problem):
```
root@ubuntu-phablet:~# lxc-info --version
2.0.11
root@ubuntu-phablet:~# waydroid container start
lxc-info: confile.c: parse_line: 2452 Unknown configuration key "lxc.pty.max"
lxc-info: parse.c: lxc_file_for_each_line: 57 Failed to parse config: lxc.pty.max = 10

Failed to load config for waydroid
Failure to retrieve information on /var/lib/waydroid/lxc:waydroid
[20:08:54] WayDroid container is
```

**Pull request merged**